### ISSUE TITLE
Tree: Support for expandNode() and collapseNode() API 

### ISSUE THREAD

**Issue by hemantg05** [infor-design/enterprise#4707]
**Is your feature request related to a problem or use case? Please describe.**
We would like to be able to expand or collapse a particular node on tree.

tree.expandNode(node)
tree.expandNode(nodeId)
OR
node.expand();

Same goes for collapse.
tree.collapseNode(node)
tree.collapseNode(nodeId)
OR
node.collapse();

**Describe alternatives you've considered**
1. We have considered toggleNode(). But it doesn't work. Imagine we need to expand a node 3 levels deep in a completely collpased tree. toggleNode() on 3 level deep node would not work.
2. We also tried.
     const node = this.tree.findById(id);
      node.data = { expanded: true }; 
      this.tree.updateNode(node);
      
      Such expanded: true doesn't work in updateNode.
      
**Additional context**
We had created an enhancement request for this API in https://github.com/infor-design/enterprise/issues/3927. But this was missed. So creating a separate enhancement request.
Infor PIM
@karinthulin, @Carlfjord

**Comment by tmcconechy** [comment]
Just checking @hemantg05 is this needed? Whats the priority for you?

**Comment by hemantg05** [comment]
@tmcconechy, yes, it is still needed. We are right now living with some dirty workaround with jquery class functions, recursive functions on tree node and toggleNode() to achieve what would be provided by this API.

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
